qa-core/src/config/internal-api/fixtures/screens.ts

```diff
@@ -0,0 +1,34 @@
+import generator from "../../generator"
+
+const randomId = generator.guid()
+
+const generateScreen = (roleId: string): any => ({
```

Comment:
I think it would be a really good idea for us to be verifying the response content as well as the status. 

We could add another function here `expectedScreen(roleId, _id, _rev)` and call it after the response completes like:
```ts
expect(response.body).toEqual(expectedScreen(roleId, screen._id, screen._rev)
```

It's likely it will share a lot of fields with the payload as in a lot of cases the api request / response is a direct passthrough to / from databases, so there could be some re-use in the `expectedScreen` function and the `generateScreen` function.

Reply:
The fields that differ between payload and response can only be verified that they exist. Besides the `createdAt`, how could we know the `_id` to expect a value, for example?

Reply:
fair point, this would only really work for updates. would probably need to check that the fields are first defined, then delete
```ts
delete screen._id
delete screen._rev
delete screen.createdAt
delete screen.updatedAt
```
this would get tedious over time doing for many responses, so you could have a function
```ts
expectDocument = (document: Document) => {
   expect(document._id).tobeDefined()
   delete document._id

   expect(document_rev).tobeDefined()
   delete document._rev

   expect(document.createdAt).tobeDefined()
   delete document.createdAt

   expect(document.updatedAt).tobeDefined()
   delete document.updatedAt
}

expectedScreen = {} // some screen object

expectScreen(screen: Screen) => {
   expectDocument(screen)
   expect(screen).toEqual(expectedScreen)
}
```